src/ipc/zone/server/actor_control.rs

```diff
@@ -127,7 +128,20 @@ pub enum ActorControlCategory {
     WalkInTriggerRelatedUnk1 { unk1: u32 },
 
     #[brw(magic = 290u32)]
-    Emote { emote: u32 },
+    Emote {
+        /// The id of the emote.
+        emote: u32,
+        /// 0/false = text shown, 1/true = text hidden
+        #[brw(pad_before = 4)]
```

Comment:
Ditto, I also wonder since this is identical to the ClientTrigger we can make a generic struct? (The enum can live there too)